.appveyor/build_libedit.bat

```diff
@@ -0,0 +1,59 @@
+@echo on
+setlocal
+
+call "%~dp0set_paths.bat"
+
+cd "%BUILDDIR%"
+
+if exist "%BINARIESDIR%\libedit" (
```

Comment:
As such there are no special instructions for developers to build PBS on windows using these scripts, you just have to run 3 steps mentioned in .appveyor.yml file but however these scripts to works you need some tools to be installed in system (like MinGW, SVN, Perl, Cmake etc... for more info check set_paths.bat) and based on path where you installed this tool modify set_paths.bat file.